task_processing/metrics.py

```diff
@@ -0,0 +1,54 @@
+import mock
+
+try:
+    import yelp_meteorite
+    METRICS_ENABLED = True
+except Exception:
+    METRICS_ENABLED = False
+
+TASKPROC_TASK_LAUNCHED_COUNT = 'taskproc_task_launched_count'
```

Comment:
This is overly verbose, I don't think you need to namespace everything by taskproc.